RIOT-OS/RIOT
# boards/nucleo-l433rc: add pinout to documentation page

### Contribution description

This PR adds pinout diagram to the `nucleo-l433rc` board.

### Testing procedure

Generate doc and see if everything is fine.

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-l433rc.html
```
### Issues/PRs references

None
